dockerbuild/idsync-cron

```diff
@@ -2,5 +2,5 @@
 # See https://crontab.guru/ for help with cron pattern.
 
 # m h dom mon dow user command
-2 0 * * * root /data/run-full-batch.sh > /proc/1/fd/1 2>&1
-*/5 * * * * root /data/run-incremental-batch.sh > /proc/1/fd/1 2>&1
+*/5 * * * * root /data/run-full-batch.sh > /proc/1/fd/1 2>&1
+#*/5 * * * * root /data/run-incremental-batch.sh > /proc/1/fd/1 2>&1
```

Comment:
I'm not too familiar with sync process, but any reason why changing full batch to every 5 minutes and removing incremental batch?

Reply:
I see commit message now, I'd probably feel safer if Matt approves though.